third-party/chpl-venv/Makefile

```diff
@@ -92,7 +92,12 @@ chpldoc-venv: install-chpldeps
 
 c2chapel-venv: install-chpldeps
 
-chapel-py-venv: install-chpldeps
+# the Python bindings depend on libChplFrontendShared.so, which we probably
+# shouldn't explicitly list here. So, make this target FORCE to always
```

Comment:
Why shouldn't we list it here?

Reply:
For one, it breaks the top-down nature of the way we try to specify our targets (though it's broken in other places, too). For another, we don't have a Makefile target we can reference (since the target is generated by CMake) -- and relying on the `.so` file's exact location and name is awkward. Either way, other files (specified in `setup.py`) are dependencies, and listing them twice (once in setup.py and once here) would be error-prone. I'll update the comment.